docs/topics/downloader-middleware.rst

```diff
@@ -190,6 +190,12 @@ The following settings can be used to configure the cookie middleware:
 * :setting:`COOKIES_ENABLED`
 * :setting:`COOKIES_DEBUG`
 
+Notice that if the :class:`~scrapy.http.Request` 
+has ``meta['dont_merge_cookies']`` present then 
```

Comment:
It can be present, but False; I expect cookies to be merged in this case. Is it how it works?

Reply:
What do you think about adding more links - from here to somewhere in Request docs, and back?